soroban-env-common/env.json

```diff
@@ -1525,11 +1525,11 @@
                     "name": "verify_sig_ed25519",
                     "args": [
                         {
-                            "name": "x",
+                            "name": "k",
                             "type": "BytesObject"
                         },
                         {
-                            "name": "k",
+                            "name": "x",
```

Comment:
Nice, looks like this matches the SDK now. https://github.com/stellar/rs-soroban-sdk/blob/main/soroban-sdk/src/crypto.rs#L33